tests/components/stream/test_recorder.py

```diff
@@ -72,7 +72,7 @@
 async def test_recorder_save():
     """Test recorder save."""
     # Setup
-    source = generate_h264_video()
+    source = open(generate_h264_video(), 'rb')
```

Comment:
We should use `with open() as source:` so the file is closed properly at the end.